Makefile

```diff
@@ -144,7 +144,7 @@ pipelines: ## Upload pipelines to Concourse
 showenv: ## Display environment information
 	$(eval export TARGET_CONCOURSE=deployer)
 	@echo CONCOURSE_IP=$$(aws ec2 describe-instances \
```

Comment:
All fine, I just wonder if we should add `export` in front of `CONCOURSE_IP` like all other env. vars produced by `showenv` do have.